### PR TITLE
Schema an color changes

### PR DESCRIPTION
**Pull request by stsiek**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Review by stsiek — COMMENTED** [review]
Please change this schema, this was a mistake. The connector isn't in use yet.

File: certified-connectors/Converter/apiProperties.json [hunk 1]
```diff
@@ -15,9 +15,9 @@
         }
       }
     },
-    "iconBrandColor": "#da3b01",
+    "iconBrandColor": "#000000",
     "capabilities": [],
-	"publisher": "Power2Apps P2A GmbH",
+	  "publisher": "Power2Apps P2A GmbH",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Converter/apiProperties.json:20]: @stsiek please fix the indentation after the changes.

**Comment by stsiek** [comment]
Please merge this commit, there was a mistake in it before. The connector isn't in use yet. Thank you.

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @stsiek,

I hope you are doing well.

Congratulations your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) using your latest commit id, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio.

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.

If your connector passes all tests, the deployment process begins, and it typically takes up to 5 to 6 weeks to deploy your connector to all of our regions.

Please let me know if you have any questions.

Thank you very much for working with us.

**Pull request merged**